app/views/admins/show_items.html.erb

```diff
@@ -1,11 +1,11 @@
-All Items | <%= link_to "Pending Items", admins_path, method: :'get' %>
+All Items | <%= link_to "Back To Admin Page", admins_path() %>
```

Comment:
You don't need to add empty parentheses to your `index` path.

Reply:
removed ().